src/js/commons/Header/index.jsx

```diff
@@ -1,11 +1,47 @@
 // ########## Import Dependencies Here ##########
 import React, { Fragment } from 'react';
-import { shape, arrayOf, string } from 'prop-types';
+// import { shape, arrayOf, string } from 'prop-types';
 
 // ########## Import Components Here ##########
 
-const Header = (props) => {
-  const { header } = props;
+const Header = () => {
+  // const { header } = props;
+  const header = {
```

Comment:
@sultanaslam Same goes here too, no data file should be within component. Keep data out of the component.